webpack.config.cjs

```diff
@@ -206,16 +207,6 @@ class EmitSubjectsSchemaPlugin {
 						execSync( './schema/build.mjs', { stdio: 'inherit' } );
 						const schema = readFileSync( SCHEMA_SRC );
 
-						// Write to WordPress plugin directory
-						await fs.promises.mkdir(
```

Comment:
We can remove the `const fs = require( 'fs' );` at the top of the file as `fs` is no longer used